### PR TITLE
Node.js week1 is ready for comments

### PR DESCRIPTION
**Pull request by aijazraja**

### CODE REVIEW

File: nodejs/week4/readme.md [hunk 1]
```diff
@@ -1 +0,0 @@
-# Readme
```
Comment (aina21) [inline nodejs/week4/readme.md:1]: this file is not related to this PR.

File: nodejs/week5/readme.md [hunk 2]
```diff
@@ -1 +0,0 @@
-# Readme
```
Comment (aina21) [inline nodejs/week5/readme.md:1]: this file as well.

File: nodejs/week1/src/backend/app.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+/* 
+/meals
+Respond with the json for all the meals. For each meal, if there are reviews matching it's meal ID, 
+add these reviews to each meal in the form of an array. For meals that do not have any reviews,
+the "reviews" property will be an empty array. (watch the GIF below to understand how it should be 
+structured)
+*/
+
+const copiedMeals = meals;
+const mealsWithReviews = copiedMeals.map((meal) => {
```
Comment (aina21) [inline nodejs/week1/src/backend/app.js]: maybe it's better to remove copiedMeals and then map on meals.

**Review by aina21 — CHANGES_REQUESTED** [review]
nice job :+1:

File: nodejs/week1/src/backend/app.js (hunk 3 above)
Comment (aijazraja) [reply nodejs/week1/src/backend/app.js]: HI Aina
I have fixed your comments in my code
